notebooks/national_run.py

```diff
@@ -44,15 +44,15 @@
 import os
 from datetime import datetime
 
+import model as mdl
```

Comment:
should this now not be

```suggestion
import nhp.model as mdl
```

Reply:
![image](https://github.com/user-attachments/assets/2cd2d413-317f-4bf3-ac29-0ccdf3f0f0f2)

Tried - didn't work. Need to set `src` or PATH somewhere in the notebook?

Reply:
I'll make this as a separate subissue. will approve for now